src/wallets/injected/SenderWallet.ts

```diff
@@ -29,13 +31,14 @@ class SenderWallet extends InjectedWallet implements ISenderWallet {
 
     const state = getState();
 
+
     this.onAccountChanged();
 
     this.onNetworkChanged();
 
     return this.wallet
       .init({ contractId: state.options.accountId })
-      .then((res) => console.log("SenderWallet:init", res));
+      .then((res) => logger.log("SenderWallet:init", res));
```

Comment:
Missing a couple of other console calls in this module

Reply:
Had them locally. Forgot to commit them. Thanks!